src/cljs/zetawar/data.cljs

```diff
@@ -133,6 +136,7 @@
                 :cost 75
                 :movement 9
                 :can-capture true
+                :can-repair false
```

Comment:
It complicates things a bit, but I think this should be a set of armor classes rather than a boolean. That way we can have engineers that can repair armored units, etc.